src/lib/common/JsonHelper.cpp

```diff
@@ -149,6 +154,46 @@ void JsonHelper::addRaw(const std::string& key, const std::string& value)
 }
 
 
+/* ****************************************************************************
+*
+* JsonHelper::addDouble -
```

Comment:
That's the idea... to add new addNumber() functions for other kind of number types if we need in the future.